meta-genivi-dev/meta-genivi-dev/recipes-dev-hmi/genivi-dev-platform-hmi/bluetoothpairingapp_git.bb

```diff
@@ -0,0 +1,39 @@
+# Copyright (C) 2017 Viktor Sjölind <viktor.sjolind@pelagicore.com>
+# Released under the MIT license (see COPYING.MIT for the terms)
+
+DESCRIPTION = "Application used to pair bluetooth devices"
+LICENSE = "MPL-2.0"
+
```

Comment:
It's good practice to [include a checksum for the license file](http://www.yoctoproject.org/docs/current/mega-manual/mega-manual.html#usingpoky-configuring-LIC_FILES_CHKSUM).